libr/include/r_anal.h

```diff
@@ -2058,7 +2058,12 @@ R_API RAnalEsilDFG *r_anal_esil_dfg_expr(RAnal *anal, RAnalEsilDFG *dfg, const c
 R_API RStrBuf *r_anal_esil_dfg_filter(RAnalEsilDFG *dfg, const char *reg);
 R_API RStrBuf *r_anal_esil_dfg_filter_expr(RAnal *anal, const char *expr, const char *reg);
 R_API RList *r_anal_types_from_fcn(RAnal *anal, RAnalFunction *fcn);
+
+R_API void enum_type_fini(void *e, void *user);
```

Comment:
Well I didn't have them exported, but I saw having the same function in 3 different modules that was using it and didn't justify the repeating to myself, so I tried having it implemented only once. The declaration is like that because the signature is mandated by RVectorFree signature

Reply:
So there are 2 ways:
1) if these functions are used only within the ranal module, but in different files, you can create a new header file, similar to `libr/bin/i/private.h` and use `R_IPI` instead of `R_API` to define these functions.
2) if you actually want to export those functions because they may be useful to other modules, keep them here, but add the `r_` prefix and change the signature to something good, not dependennt on RVectorFree. If you then need this function inside RVectorFree you will have to create a small wrapper.

Reply:
Well so far they are used in any module working with BaseTypes, so in any anal/type*.c module. I'll probably pick the header way then

Reply:
perfect :+1: